tasks/main-blivet.yml

```diff
@@ -161,6 +161,33 @@
     daemon_reload: true
   when: blivet_output['mounts']
 
+- name: Set directory owner
+  command: "chown {{ mount_info['owner'] }} {{ mount_info['path'] }}"
```

Comment:
Please use the `file` module to manage ownership/permissions idempotently and performantly:
```yaml
- name: Manage mount ownership/permissions
  file:
    path: "{{ mount_info['path'] }}"
    owner: "{{ mount_info['owner'] if 'owner' in mount_info else omit }}"
    group: "{{ mount_info['group'] if 'group' in mount_info else omit }}"
    mode: "{{ mount_info['mode'] if 'mode' in mount_info else omit }}"
```
I would also recommend using `mode` instead of `permissions` to be more consistent with other Ansible modules and other system roles that have a similar feature.

Reply:
Thanks for the suggestion. Modified.